classes/plugins/PluginHelper.inc.php

```diff
@@ -103,13 +106,14 @@ function installPlugin($path, &$errorMsg) {
 		$versionDao = DAORegistry::getDAO('VersionDAO'); /* @var $versionDao VersionDAO */
 		$installedPlugin = $versionDao->getCurrentVersion($pluginVersion->getProductType(), $pluginVersion->getProduct(), true);
 
+		$fileManager = new FileManager();
 		if (!$installedPlugin) {
 			$pluginLibDest = Core::getBaseDir() . '/' . PKP_LIB_PATH . '/' . strtr($pluginVersion->getProductType(), '.', '/') . '/' . $pluginVersion->getProduct();
 			$pluginDest = Core::getBaseDir() . '/' . strtr($pluginVersion->getProductType(), '.', '/') . '/' . $pluginVersion->getProduct();
 
 			// Copy the plug-in from the temporary folder to the
 			// target folder.
-			$this->_cutAndPasteTemporaryFile($path, $pluginDest, $pluginLibDest, $errorMsg);
+			$this->_cutAndPasteTemporaryFile($path, $pluginDest, $pluginLibDest, $errorMsg, $fileManager);
```

Comment:
Shouldn't here and at the other place where this function is called the `return null` be considered e.g. in case there was an error when copying the file, like in the original version?